website/docs/en/guide/basic/output-files.mdx

```diff
@@ -38,7 +38,7 @@ In the filename, `[name]` represents the entry name for this file, such as `inde
 
 ## Development mode output
 
-In development mode, Rsbuild stores build outputs in memory on the dev server by default, rather than writing them to disk. This reduces filesystem operation overhead. You can refer to [View Static Assets](/guide/basic/server#view-static-assets) to view all static assets generated in the current build.
+In development mode, Rsbuild stores build outputs in memory via the dev server by default, rather than writing them to disk. This reduces filesystem operation overhead. You can refer to [View Static Assets](/guide/basic/server#view-static-assets) to view all static assets generated in the current build.
```

Comment:
```suggestion
In development mode, Rsbuild stores build outputs in memory on the dev server by default, rather than writing them to disk. This reduces filesystem operation overhead. You can refer to [View Static Assets](/guide/basic/server#view-static-assets) to view all static assets generated in the current build.
```